src/sendStatus.ts

```diff
@@ -0,0 +1,37 @@
+export class SendStatus {
+    private readonly deviceId: string;
+    private succeed: number;
+    private failed: number;
+    private readonly total: number;
+
+    constructor(deviceId: string, total: number) {
+        this.deviceId = deviceId;
+        this.succeed = 0;
+        this.failed = 0;
+        this.total = total;
+    }
+
+    public getDeviceId(): string {
+        return this.deviceId;
+    }
+    public getSucceed(): number {
```

Comment:
consider expose properties instead of adding functions, since the property can be read and edited